examples/basic/README.md

```diff
@@ -0,0 +1,78 @@
+# A Basic TiDB cluster with monitoring
+
+> **Note:**
+>
+> This setup is for test or demo purpose only and **IS NOT** applicable for critical environment. Refer to the [Documents](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/prerequisites/) for production setup.
+
+The following steps will create a TiDB cluster with monitoring, the monitoring data is not persisted by default.
+
+**Prerequisites**: 
+- Has TiDB operator `v1.1.0-beta.1` or higher version installed. [Doc](https://pingcap.com/docs/stable/tidb-in-kubernetes/deploy/tidb-operator/)
+- Has default `StorageClass` configured, and there are enough PVs (by default, 6 PVs are required) of that storageClass:
+  
+  This could by verified by the following command:
+  
+  ```bash
+  > kubectl get storageclass
+  ```
+  
+  The output is similar to this:
+  
+  ```bash
+  NAME                 PROVISIONER               AGE
+  standard (default)   kubernetes.io/gce-pd      1d
+  gold                 kubernetes.io/gce-pd      1d
+  ```
+  
+  Alternatively, you could specify the storageClass explicitly by modifying `tidb-cluster.yaml`.
+
+## Install
+
+The following commands is assumed to be executed in this directory.
+
+Install the cluster:
+
+```bash
+> kubectl -n <namespace> apply -f ./
+```
+
+Wait for cluster Pods ready:
+
+```bash
+watch kubectl -n <namespace> get pod
+```
+
+It is expected to see the following outputs:
+
+```bash
+```
+
+## Explore
+
+Explore the TiDB sql interface:
+
+```bash
+> kubectl -n <namespace> port-forward svc/basic-tidb 4000:4000 &>/tmp/pf-tidb.log &
+> mysql -h localhost:4000 -u root
```

Comment:
I need to run `mysql -h 127.0.0.1 -P 4000` to connect to the TiDB server

```
root@tidb-operator-dev:/go/src/github.com/pingcap/tidb-operator# mysql -h localhost:4000 -u root
ERROR 2005 (HY000): Unknown MySQL server host 'localhost:4000' (-2)
root@tidb-operator-dev:/go/src/github.com/pingcap/tidb-operator# mysql -h localhost -P 4000
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
root@tidb-operator-dev:/go/src/github.com/pingcap/tidb-operator# mysql -h 127.0.0.1 -P 4000
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MySQL connection id is 2
Server version: 5.7.25-TiDB-v3.0.8 MySQL Community Server (Apache License 2.0)

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MySQL [(none)]> select tidb_version();
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| tidb_version()                                                                                                                                                                                                                                                                                       |
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Release Version: v3.0.8
Git Commit Hash: 8f13cf1449bd8903ff465a4f12ed89ecbac858a4
Git Branch: HEAD
UTC Build Time: 2019-12-31 11:14:59
GoVersion: go version go1.13 linux/amd64
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false |
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.002 sec)

MySQL [(none)]> 
```

Reply:
good catch